.github/workflows/py-tests.yml

```diff
@@ -12,12 +12,12 @@
 name: py-tests
 on:
   push:
-    branches: [main]
+    branches: [main, '0.[0-9]+.[0-9]+']
```

Comment:
One quick enhancement suggestion here, let's follow a standard yaml pattern with below snippet everywhere in CI files!
```
branches:
      - main
      - '0.[0-9]+.[0-9]+'
```